wai-aria/role/table-roles.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Table Role Verification Tests</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+  </head>
+  <body>
+
+  <p>Tests <a href="https://w3c.github.io/aria/#table">table</a> and related roles.</p>
+
+<script>
+AriaUtils.verifyRolesBySelector(".ex");
+AriaUtils.verifyGenericRolesBySelector(".ex-generic");
```

Comment:
Won't this throw an error since there aren't any tests?

Reply:
```
ERROR message: Selector passed in verifyRolesBySelector(".ex") should match at least one element.
```

https://wpt.fyi/results/wai-aria/role/table-roles.html?diff&filter=ADC&run_id=5131185556029440&run_id=4850203527479296